crates/biome_js_analyze/src/semantic_analyzers/correctness/use_exhaustive_dependencies.rs

```diff
@@ -294,7 +283,13 @@ impl VisitNode<JsonLanguage> for Hooks {
                 self.dependencies_index =
                     self.map_to_usize(&value, name_text, usize::MAX, diagnostics);
             }
-            _ => {}
+            _ => {
+                report_unknown_member_name(
+                    &name,
+                    &["name", "closureIndex", "dependenciesIndex"],
+                    diagnostics,
+                );
```

Comment:
Moving the arrays outside of a const doesn't seem necessary, can we revert it please?